### ISSUE TITLE
[BUG] 1942 (both versions) game screen is black

### ISSUE THREAD

**Issue by mrMuppet** [markus-zzz/myc64-pocket#18]
When is start the game 1942 the title screen seems ok but has a flickering square in the upper right and hen the game is started, only the upper 10% are displayed. The rest of the screen is black.

![PXL_20240602_112244086](https://github.com/markus-zzz/myc64-pocket/assets/2955497/3d854533-8f05-4713-9376-e6e287b119a9)

**Comment by markus-zzz** [comment]
Looking at VICE emulator, the title screen of the game (without any interaction whatsoever) is supposed to alternate between the following two
![vice-screen-20240602214248](https://github.com/markus-zzz/myc64-pocket/assets/8939552/d50caab9-5137-4470-a1c5-f89bd92dc505)
![vice-screen-20240602214220](https://github.com/markus-zzz/myc64-pocket/assets/8939552/26b3267d-c71c-46ca-a53a-87f18fd65c5e)
On the pocket only the former appears OK and the latter is truncated as in the screenshot in previous comment.

**Comment by markus-zzz** [comment]
Three screen dumps captured from simulation, showing both the flickering reported as well as the truncation.

![vicii-1024](https://github.com/markus-zzz/myc64-pocket/assets/8939552/18c7a37e-c8b0-4bc9-928d-bc01dd4a55db)
![vicii-1025](https://github.com/markus-zzz/myc64-pocket/assets/8939552/d0b1f49f-0d43-4e52-a1e6-2ce0b032e7e8)
![vicii-1033](https://github.com/markus-zzz/myc64-pocket/assets/8939552/94a3c606-d542-4b4f-bdbc-8ad2dd9729de)

**Comment by markus-zzz** [comment]
The game sets `$d011` to zero when rendering the table part of the high score screen, this among other things sets 'display enable' to zero. According to my understanding that means that the border color should cover the entire screen. However if I change the RTL to ignore the 'display enable' bit the screen renders as below

![vicii-1060](https://github.com/markus-zzz/myc64-pocket/assets/8939552/747f7e6c-b125-4d6a-a3fe-a87dd6488f7a)

Both horizontal and vertical sprite doubling is enabled but since that feature is not yet implemented the text appears tiny.

**Comment by markus-zzz** [comment]
The missing 'Coding & Graphics' footer is because of

```
    with m.If(r_d011[3]):
      m.d.comb += [display_window_y.eq((raster >= 0x33) & (raster <= 0xfa))]
    with m.Else():
      m.d.comb += [display_window_y.eq((raster >= 0x37) & (raster <= 0xf6))]
```
where we instead should check for equality so that the 'turn off lower border trick' described [here](https://retrocomputing.stackexchange.com/questions/2598/what-graphical-effects-can-be-created-by-changing-the-vic-ii-settings) would work.

**Comment by markus-zzz** [comment]
Should be resolved as of release `v0.2.2`.

There are some, in my opinion, minor graphical glitches left but I feel that they should be dealt with when doing systematic testing of the VIC-II comparing pixel to pixel with a known good reference such as VICE emulator.
